specification/monitor/resource-manager/readme.md

```diff
@@ -347,6 +347,16 @@ input-file:
 - Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json
 ```
 
+### Tag: package-2019-11-preview-only
```

Comment:
Are you going to support some sdks like Python/Java? If so, we might need to change `readme.python.md` as well.

Reply:
@MyronFanQiu - we plan to work on SDKs in a 2-3 months - we are are expecting a couple of iterative changes to the API early in public preview and we want them to stabilize first

Are we OK to merge this PR now? If yes, could you do that?